test/integration/etcd/data.go

```diff
@@ -533,16 +626,40 @@ func GetEtcdStorageDataForNamespaceServedAt(namespace string, version string, re
 			}
 		}
 	}
+	validateStorageData(etcdStorageData)
 	return etcdStorageData
 }
 
+func validateStorageData(etcdStorageData map[schema.GroupVersionResource]StorageData) {
+	exceptions := map[schema.GroupVersionResource]bool{
+		gvr("internal.apiserver.k8s.io", "v1alpha1", "storageversions"): true,
```

Comment:
This resource does not have proper lifecycle information. Adding lifecycle information would imply a removedVersion being set but this alpha API has been around for years already and I'm not sure if it's something that can be easily set.